.github/workflows/continuous-integration.yml

```diff
@@ -42,15 +42,20 @@ jobs:
           MOCHA_FILE: test-results/unit-tests.xml
 
       - name: Generate coverage report by app
-        run: node script/app-coverage-report.js
+        run: node script/app-coverage-report.js > test-results/coverage_report.txt
+
+      - name: Get codecoverage
+        id: codecoverage
+        run: echo ::set-output name=MARKDOWN::$(node ./script/github-actions/codecoverage-format-report.js)
```

Comment:
```suggestion
      - name: Get code coverage
        id: code-coverage
        run: echo ::set-output name=MARKDOWN::$(node ./script/github-actions/code-coverage-format-report.js)
```

Reply:
Is L49 support to change `summary: ${{ steps.codecoverage.outputs.MARKDOWN }}` or did you want me to rename file?

Reply:
Let's rename the file as well.